docs/src/references/c8y-configuration-management.md

```diff
@@ -162,4 +173,32 @@ Note that:
   for instance `tedge/configuration_change//etc/tedge/mosquitto-conf/c8y-bridge.conf`.
 * Since the type of configuration file is used as an MQTT topic name, the characters `#` and `+` cannot be used in a type name.
   If such a character is used in a type name (or in the path of a configuration file without explicit type),
-  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
\ No newline at end of file
+  then the whole plugin configuration `/etc/tedge/c8y/c8y-configuration-plugin.toml` is considered ill-formed.
+  
+## Configuration via MQTT
+
+The configuration of the `c8y_configuration_plugin` (i.E. the content of its TOML file) is additionaly provided on MQTT. That allows adding configurations without need for local filesystem access (e.g. for external devices or containers).
```

Comment:
* What is the motivation for this?
* A key feature of the `c8y_configuration_plugin` is to configure the plugin itself from the cloud. One must keep this feature working.
* Composing a file from fragments is indeed a problem thin-edge should address. But this cannot be done in an ad-hoc manner for each kind of file. [A discussion is already open on that topic](https://github.com/thin-edge/thin-edge.io/discussions/1199).

Reply:
> * What is the motivation for this?
> 
> * A key feature of the `c8y_configuration_plugin` is to configure the plugin itself from the cloud. One must keep this feature working.

The motivation behind that is to enable external devices or containers to add their configuration files by their own to the C8Y configuration plugin. However as you say it is always possible to add a config files from the cloud to the C8Y configuration plugin.

So based further requirements analsyis we might able to deprioritize that topic, and for now focus on consuming/providing configuration files from/to external devices/containers.